update-python-kata/README.md

```diff
@@ -0,0 +1,43 @@
+# FAQ: Update Python kata
+
+## What is the problem with old Python kata?
+
+The problem described here is caused by the fact that test suites of old kata do not conform to the form expected by Codewars code runner and produce test output in a format not conforming to the expected one. The main reason for that is that calls to assertions (`assert_equals`, `expect`, etc.) are not placed inside of an `@it` block, but in a scope of a `@describe` block or just in the top-level scope of tests.
+
+Additionally, missing imports for test framework and solution modules make some additional preprocessing necessary and potentially mess up line numbers in error messages.
+
+## Why was everything OK until now, and now the problem suddenly popped up?
+
+Because of future compatibility and maintenance. Python is one of the oldest supported languages on Codewars and it drags behind a large bag of compatibility hacks and workarounds. The test output is handled in the runner by a mechanism that is not related to any programming language, and workarounds needed to make Python tests work affect the maintainability of the runner overall. Test output for the majority of languages is well defined and structured, except a few which require a large set of workarounds, including Python.
+
+Fixing tests will allow for cleaning up the code of the runner and will make it easier to develop it, provide new features, and introduce new languages and language versions in the future.
+
+Additionally, from the user perspective, improperly structured tests tend to result in poor output and are prone to buggy display in test output panel. Updating the kata can potentially fix erlated problems.
+
+## My kata work, do they have to be updated?
+
+They should be. Until they are, the "old" format of the test output has to be supported, impeding updates to Python versions or test frameworks.
+
+## Do I have to update my kata by myself?
+
+No. It would be nice if Python authors and translators took the responsibility for fixing kata and translations they created, but they are not forced to. However, authors should allow kata to get fixed by someone else. There's a group of users willing to fix affected kata, and they use available means to track them and update them. If you do not want to update kata on your own, let them do their work. Do not mark their issues as resolved if they are not, and do not obstruct their work.
+
+## Will my kata stop working if they are not updated?
+
+No. The Codewars staff explicitly take care to not leave kata broken when updating the code runner. When any change is going to be introduced, it will be done in a backwrds compatible way. However, if your kata would not work without being fixed, it will simply postpone the update or increase the efforts needed to roll one out.
```

Comment:
```suggestion
No. The Codewars staff explicitly take care to not leave kata broken when updating the code runner. When any change is going to be introduced, it will be done in a backwards compatible manner. However, if your kata would not work without being fixed, it will simply postpone the update or increase the efforts needed to roll one out.
```